scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/job/InternalJob.java

```diff
@@ -693,9 +385,396 @@ public ChangedTasksInfo terminateTask(boolean errorOccurred, TaskId taskId,
 
         return changesInfo;
     }
+    
```

Comment:
is there so many code changes or is it a formatting problem ? It's too difficult to review here.